components/Header.tsx

```diff
@@ -21,7 +21,6 @@ const Header: React.FC = () => {
       <nav>
         <Navbar />
       </nav>
-
     </ThemeProvider>
```

Comment:
Wasn't made in this commit but same here, probably not necessary as it's in the app